packages/common/driver-definitions/src/storage.ts

```diff
@@ -256,6 +256,11 @@ export interface IDocumentDeltaConnection extends IDisposable, IEventProvider<ID
      * Submit a new signal to the server
      */
     submitSignal(message: any): void;
+
+    /**
+     * TODO
+     */
+    processInboundMessages?(messages: ISequencedDocumentMessage[]): ISequencedDocumentMessage[];
```

Comment:
i'm not totally sure why we need a new method here. Based on the discussion in the issue its not clear what the design here is. It would be good to articulate the design in the description so its available for open source.